docs/node_scenarios.md

```diff
@@ -28,9 +28,9 @@ A google service account is required to give proper authentication to GCP for no
  
 After creating the service account you'll need to enable the account using the following: ```export GOOGLE_APPLICATION_CREDENTIALS="<serviceaccount.json>"```
 
-**NOTE**: The `stop_start_kubelet_scenario` and `node_crash_scenario` scenarios are supported as they are independent of the cloud platform.
+**NOTE**: The `node_crash_scenario` scenario is supported independent of the cloud platform.
```

Comment:
Do you think we need to flag the scenario as not-applicable to the generic cloud_provider in the code.  i.e. treat it like reboot, etc?   Sorry to be so nitpicky about this

Reply:
No worries! I was thinking abut that as well just to maybe inform the user of their incorrect scenario set up.

Reply:
Added a lot of if statements here to log what is happening. Is this over kill?

Reply:
Found that the doc for generic scenarios should've said "stop_kubelet_scenario" instead of stop/start. Updated this